assets/css/js_interop.css

```
@@ -116,7 +116,7 @@ solely client-side operations.
 }
 
 [data-el-outline-item][data-js-is-viewed] {
-  @apply text-gray-900;
+  @apply text-gray-950;
```

We don't use 950 anywhere (we don't even have it in the palette), so I would stick to 900 :)

```suggestion
  @apply text-gray-900;
```